sys/shell/commands/sc_at30tse75x.c

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2015 Daniel Krebs
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_shell_commands
+ * @{
+ *
+ * @file
+ * @brief       Provides shell commands to test AT30TSE75x temperature sensor
+ *
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include "at30tse75x.h"
+#define MODULE_AT30TSE75X
```

Comment:
This slipped through by accident. Eclipse doesn't know those defines, so auto-completion doesn't work inside unknown `#ifdef`s.